unittests/web/ServerTests.cpp

```diff
@@ -359,3 +359,76 @@ TEST_F(WebServerTest, WsPayloadOverload)
         res,
         R"({"payload":"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa","warning":"load","warnings":[{"id":2003,"message":"You are about to be rate limited"}]})");
 }
+
+constexpr static auto JSONServerConfigWithAdminPassword = R"JSON(
```

Comment:
Let's try to keep `const` and `constexpr` on east side? 🥇 
Ideally this would be `static auto constexpr` but we have all styles mixed atm. 
We can change them all in one separate PR perhaps.

Reply:
We should move to a newer clang-format so this could be forced
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#qualifieralignment

#848